packages/arg-parser/.depcheckrc

```diff
@@ -7,3 +7,5 @@ ignores:
  - eslint-config-mongodb-js
 # needed as a peer dependency of @mongodb-js/devtools-connect
  - mongodb
+ignore-patterns:
+ - .eslintrc.js
```

Comment:
depcheck's special eslint code seemed to get confused by our usage of `extends`. The simplest workaround I could come up with was to stick this in every `.depcheckrc`.